internal/config/config.go

```diff
@@ -155,7 +155,7 @@ type AzureConfig struct {
 	//   Deploy Azure Disk CSI driver with on-node encryption. For details see: https://docs.edgeless.systems/constellation/architecture/encrypted-storage
 	DeployCSIDriver *bool `yaml:"deployCSIDriver" validate:"required"`
 	// description: |
-	//   Use Confidential VMs. If set to false, Trusted Launch VMs are used instead. See: https://docs.microsoft.com/en-us/azure/confidential-computing/confidential-vm-overview
+	//   Use Confidential VMs. If set to false, Trusted Launch VMs are used instead. Trusted Launch support is experimental.
```

Comment:
Should I then also add validation to this field such that it fails if not true?

Reply:
> Should I then also add validation to this field such that it fails if not true?

I don't have an opinion on this. I can see this go both ways: 
* better for user who don't read the comments
* harder for us if we need to test / demo trusted launch to someone?

Since I don't know if any customer expressed interest in trusted launch, I can't really judge which one is more likely.